JSUserRegistrationService/src/main/java/com/example/jsuserregistrationservice/PersonalDetails.java

```diff
@@ -0,0 +1,43 @@
+package com.example.jsuserregistrationservice;
+
+import javax.validation.constraints.Min;
+import javax.validation.constraints.NotNull;
+
+/**
+ * Copyright (c) 2016, 2017, Jumia.
+ */
+public class PersonalDetails {
+
+    @NotNull
```

Comment:
This will accept empty names "" and as such this annotation should be @NotEmpty or @NotBlank instead.

Reply:
done